components/SeasonTabs.vue

```diff
@@ -65,28 +63,15 @@ export default Vue.extend({
       }
     };
   },
+  computed: mapState({
+    seasons: (state) => state.tvShows.seasons,
+    seasonEpisodes: (state) => state.tvShows.seasonEpisodes
+  }),
```

Comment:
Any reason for using this instead of something like this (Which is what I usually see used) ?
```typescript
...mapState('tvShows', ['seasons', 'seasonEpisodes'])
```

Reply:
Yea, type safety